.github/workflows/tests.yml

```diff
@@ -18,18 +18,26 @@ jobs:
     strategy:
       matrix:
         include:
-          - php-version: 7.2
+          - php-version: 7.4
```

Comment:
you should not remove this PHP version that we continue to support

Reply:
It's revert